plugins/dialogui/plugin.js

```diff
@@ -1249,6 +1254,41 @@ CKEDITOR.plugins.add( 'dialogui', {
 
 		/** @class CKEDITOR.ui.dialog.radio */
 		CKEDITOR.ui.dialog.radio.prototype = CKEDITOR.tools.extend( new CKEDITOR.ui.dialog.uiElement(), {
+			focus: function() {
+				var children = this._.children,
+					// Focus the first radio button in the group by default.
+					focusTarget = children[ 0 ],
+					dialogInternal = this._.dialog._,
+					currentFocusIndex = dialogInternal.currentFocusIndex,
+					isMovingBackwards = currentFocusIndex > this.focusIndex,
+					isStartingNewLoop = currentFocusIndex === dialogInternal.focusList.length - 1 && this.focusIndex === 0;
+
+				// If focus was changed by using SHIFT + TAB key and the previous radio group
+				// does not have any checked element, focus the last one in the current radio group.
+				// The 'isStartingNewLoop' matters, because in the case when the dialog is shown and the
+				// first element is not focused, then, by default the 'currentFocusIndex' is sets as
+				// the last element from the focus list, and it can be treated as moving focus backwards,
+				// causing that the focus to be incorrectly set to the last element of the radio
+				// group, instead of the first one (#439).
```

Comment:
```suggestion
				// If focus was changed by using SHIFT + TAB key and the previous radio group
				// does not have any checked element, focus on the last one in the current radio group.
				//
				// When the dialog is shown and the first element is not focused,
				// then, by default the 'currentFocusIndex' is set as
				// the last element from the focus list, and it can be treated as moving focus backward,
				// causing the focus to be incorrectly set to the last element of the radio
				// group, instead of the first one (#439).
```